### ISSUE TITLE
using spherical images

### ISSUE THREAD

**Issue by monschine** [harish-vnkt/structure-from-motion#1]
Hi would be easy to adapt the code to use a spherical camera ?

Thanks

**Comment by harish-vnkt** [comment]
I don't think so. The code only reads a pre-computed camera matrix from a file and uses it for triangulation, I do not know if the process is different for a spherical camera. Besides, this repo also doesn't have a bundle adjustment, so I'd suggest you go with COLMAP.
